### ISSUE TITLE
Menu items and sub items are not displayed in correct order in front end after drag-and-drop in back end

### ISSUE THREAD

**Issue by CodeOfConduct** [joomla/joomla-cms#26956]
### Steps to reproduce the issue
In back end:
a) create (main) menu items and sub items
b) drag-and-drop menu items and sub items to other positions in the menu

### Expected result

Sequence of menu (sub) items is the same in the front end as in the back end.

### Actual result

The sequence of the menu (sub) items in the front end is not the same as in the back end for some items that have been dragged and dropped.

### System information (as much as possible)

Joomla! 3.9.12, Rockettheme Flux template (Gantry 5 framework), PHP 7.3

**Comment by richard67** [comment]
@CodeOfConduct You know that drag and drop ordering changes only the order within the same menu level but not moves an item e.g. from a "submenu" to a higher level?

**Comment by CodeOfConduct** [comment]
@richard67 Yes, I am aware of that. This ticket is, however, not about problems with dragging-and-dropping menu (sub) items in the back end (e.g. to a higher or lower level), but about the fact that after successfully dragging and dropping (at the same level), the sequence of menu (sub) items in the front end does not match the sequence in the back end.

**Comment by richard67** [comment]
@CodeOfConduct Can you reproduce that with the default template "Protostar", too? And maybe provide some screenshots or details? I can't reproduce that here.

**Comment by CodeOfConduct** [comment]
@richard67 In response to your request I changed the default template to Protostar and gave the menu module with which I experienced the problems position 3 in the Protostar template. The result was conclusive: in Protostar, all menu (sub) item positions in the front matched those in the back end. So, it looks as if Gantry 5 or the template do not pick up the sequence correctly. Sorry, I should have done this test before... Thanks for your suggestion. I'll close this ticket now.
